test/helper.rb

```diff
@@ -151,6 +159,8 @@ def stub_api_nodes
                'Host' => 'node.fakedestination.com:10255'
              }
            )
+           .to_timeout.then.to_timeout
```

Comment:
There is duplicate chaining here. Do we need it?

Reply:
Not sure to be honest, I start adding to_timeout to mocks until test fails, I've seen that nodes/ is being called in 2 places at least hence I added timeout twice.